cdk/bin/cdk.ts

```diff
@@ -4,4 +4,40 @@ import { Frontend } from "../lib/frontend";
 
 const app = new App();
 const cloudFormationStackName = process.env.GU_CFN_STACK_NAME;
-new Frontend(app, "Frontend", { stack: "support", cloudFormationStackName });
+
+new Frontend(app, "Frontend-PROD", {
```

Comment:
I followed the example in [this PR](https://github.com/guardian/amiable/pull/117) for upgrading to CDK 39. They pulled out all of the stage dependent values as props to the stack, which is what I've done here